tools/Azure.Mcp.Tools.Monitor/src/Commands/ActivityLog/ActivityLogListCommand.cs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Net;
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Core.Commands.Subscription;
+using Azure.Mcp.Core.Models.Option;
+using Azure.Mcp.Core.Options;
+using Azure.Mcp.Tools.Monitor.Commands;
+using Azure.Mcp.Tools.Monitor.Models.ActivityLog;
+using Azure.Mcp.Tools.Monitor.Options.ActivityLog;
+using Azure.Mcp.Tools.Monitor.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.Monitor.Commands.ActivityLog;
+
+public sealed class ActivityLogListCommand(ILogger<ActivityLogListCommand> logger)
+    : SubscriptionCommand<ActivityLogListOptions>
+{
+    private const string CommandTitle = "List Activity Logs";
+    private readonly ILogger<ActivityLogListCommand> _logger = logger;
+
+    // Define options from OptionDefinitions
+    private readonly Option<string> _resourceNameOption = ActivityLogOptionDefinitions.ResourceName;
+    private readonly Option<string> _resourceTypeOption = ActivityLogOptionDefinitions.ResourceType;
+    private readonly Option<double> _hoursOption = ActivityLogOptionDefinitions.Hours;
+    private readonly Option<ActivityLogEventLevel?> _eventLevelOption = ActivityLogOptionDefinitions.EventLevel;
+    private readonly Option<int> _topOption = ActivityLogOptionDefinitions.Top;
+
+    public override string Name => "list";
+
+    public override string Description =>
+        """
+        Always use this tool if user is asking for activity logs for a resource.
+        Lists activity logs for the specified Azure resource over the given prior number of hours.
+        This command retrieves activity logs to help understand resource deployment history, modification activities, and access patterns.
+        Returns activity log events with details including timestamp, operation name, status, and caller information. should be called to help retrieve information about why a resource failed to deploy or may not be working.
+          Required options:
```

Comment:
It might be redundant to describe the paramters here because the registered Options will have such information.